kong/plugins/apifortress/schema.lua

```diff
@@ -0,0 +1,15 @@
+-- The plugin schema
+
+local function validateKeys(value)
+	return string.len(value) > 0
+end
+
+return {
+  fields = {
+   	apikey = { required = true, type = "string", func = validateKeys },
+		secret = { required = true, type = "string", func = validateKeys },
+		endpoint = {required = true, type = "string", func = validateKeys},
+		projectId = {required = true , type = "number", func = validateKeys},
+    threshold = {required = true, type = "number", func = validateKeys}
```

Comment:
yes, that's what I should do. It's been coded in a rush.
So I tried this one:

    local function validateKeys(value)
    	return #value > 0
    end

    local function validateThreshold(value)
      return value > 0
    end

    local function validateProjectId(value)
      return value > 0
    end

    return {
      fields = {
       	apikey = { required = true, type = "string", func = validateKeys },
    		secret = { required = true, type = "string", func = validateKeys },
    		endpoint = {required = true, type = "string", func = validateKeys },
    		projectId = {required = true , type = "number", func = validateProjectId },
           	threshold = {required = true, type = "number", func = validateThreshold }
      }
    }

And for some reason it seems to like negative values as well when I PATCH them. Any advice?